Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel_test.py

```diff
@@ -1255,6 +1255,31 @@ def test_process_incidents(self, args, client, expected_result):
         assert next_run.get('last_fetch_ids') == expected_result.get('last_fetch_ids')
         assert next_run.get('last_incident_number') == expected_result.get('last_incident_number')
 
+    def test_last_run_in_fetch_incidents(self, mocker):
+        """
+        Given: - AzureSentinel client, last_fetched_ids array, last_run, first_fetch_time, and a minimum severity.
+
+        When:
+            - Calling the fetch_incidents command.
+
+        Then:
+            - Validate the call_args had the correct filter.
```

Comment:
Let's be a bit more specific here: what exactly is the last_fetched_ids (empty list), and what exactly are we expecting to see in the filter